hyperdome_client/hyperdome_client.py

```diff
@@ -33,19 +33,25 @@
 from .server_status import ServerStatus
 from .add_server_dialog import AddServerDialog
 
-import requests
 
+import socks
+import socket
+import requests
 
-session = requests.Session()
-# session.proxies = {'http': 'socks5://127.0.0.1:9050',
-#                    'https': 'socks5://127.0.0.1:9050'}
 
 class Server(object):
     def __init__(self, url='', uname='', passwd='', is_therapist=False):
+
         self.url = url
+        self._check_url()
+        print(self.url)
         self.username = uname
         self.password = passwd
         self.is_therapist = is_therapist
+    
+    def _check_url(self):
+        if self.url.find('http://') is -1 and self.url.find('https://') is -1:
```

Comment:
```suggestion
        if not self.url.startswith('http://') and not self.url.startswith('https://'):
```
Tiny bit faster and cleaner